src/ramses_tx/fingerprints.py

```diff
@@ -43,6 +43,7 @@
     "0001001B371B01FEFF": ("FAN", "37", "2019-08-29", "CVE-RF"),  # .                       31D9, 31DA
     "0001001B381B01FEFF": ("FAN", "37", "2020-02-14", "CVE-RF"),  # .                       31D9, 31DA (and I|042F, I|3120)
     "0001001B391B01FEFF": ("FAN", "37", "2021-11-04", "CVE-RF"),
+    "0001C8830C0A65FEFF": ("FAN", "37", "2020-12-17'", "VMD-07RPS13"), # .                   ClimaRad Ventura
```

Comment:
Please provide the corresponding `10E0` packet in:
 - **tests/tests/parsers/code_10e0.log**
 
 Also for the other signatures, below.